src/button-card.ts

```diff
@@ -142,6 +157,27 @@ class ButtonCard extends LitElement {
     return !!this._hass && !!this._config && this.isConnected;
   }
 
+  static getStubConfig(hass: HomeAssistant, entities: string[], entitiesFallback: string[]): ExternalButtonCardConfig {
+    const maxEntities = 1;
+    const foundEntities = findEntities(hass, maxEntities, entities, entitiesFallback, ['light', 'switch']);
+    if (llCreateCardIsSection) {
+      return {
+        entity: foundEntities[0] || '',
+        section_mode: true,
+        grid_options: {
+          rows: 2,
+          columns: 6,
+          min_rows: 1,
+          min_columns: 1,
```

Comment:
Those 2 should be removed from the card configuration, it's just useful in getGridOptions.

Reply:
Will do